### DISCUSSION TITLE
Dedicated module for test runner reporters

### DISCUSSION THREAD

**Discussion by edoardocavazza** [modernweb-dev/web#1958] [Ideas]
Hello!

As plugins author, it would be great to have a dedicated module for Test Runner reporters with exported helper methods like `reportBrowserLogs`. Is there any plan for that? Is something you would consider?

**Comment by bennypowers** [comment]
something like [this](https://modern-web.dev/docs/test-runner/reporters/write-your-own/)?

**Reply by edoardocavazza** [reply]
Not really. At the moment I am able to create custom reporters, but I would like to re-use helpers like `getTestProgress` (for the progress bar), `reportTestFileErrors` and [others](https://github.com/modernweb-dev/web/tree/master/packages/test-runner/src/reporter) that are very well designed.

**Reply by bennypowers** [reply]
So you want to add a module which exports various helpers? sounds good. We'd like to review that PR

**Reply by edoardocavazza** [reply]
Thanks! Here we go https://github.com/modernweb-dev/web/pull/1959 :)
